### PR TITLE
[Notify] Optional icon url

### PR DESCRIPTION
**Pull request by flypaper0**
# Description

Resolves # (issue)

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

File: Example/PNDecryptionService/NotificationService.swift [hunk 1]
```diff
@@ -62,57 +62,62 @@ class NotificationService: UNNotificationServiceExtension {
...
+
+            incomingMessagingIntent.setImage(senderAvatar, forParameterNamed: \.sender)
+
+            let interaction = INInteraction(intent: incomingMessagingIntent, response: nil)
+            interaction.direction = .incoming
+            interaction.donate(completion: nil)
+
+            return try content.updating(from: incomingMessagingIntent)
+        } 
+        catch {
```
Comment (chris13524) [inline Example/PNDecryptionService/NotificationService.swift:118]: This feels like a hack to wrap the whole thing in a try/catch. Can't you check for `""` and then disable the logic?

Comment (flypaper0) [reply Example/PNDecryptionService/NotificationService.swift:118]: Execution also can failed on: 
1. downloading resource (network problem)
2. accessing device storage

Before that fix all of that cases failing message decoding

Comment (flypaper0) [reply Example/PNDecryptionService/NotificationService.swift:118]: Nice that we found that

**Review by chris13524 — COMMENTED** [review]
LGTM

**Review by llbartekll — APPROVED** [review]

**Pull request merged**